test/formats/test_asf.py

```diff
@@ -55,7 +55,6 @@ def test_supports_tag(self):
             self.assertTrue(fmt.supports_tag('djmixer'))
             self.assertTrue(fmt.supports_tag('discnumber'))
             self.assertTrue(fmt.supports_tag('lyrics:lead'))
-            self.assertTrue(fmt.supports_tag('~length'))
```

Comment:
I removed this because this already gets checked in the test at https://github.com/metabrainz/picard/pull/1933/files/a7f0a27f929eed2894ac3e0e8df839576f1acbaa#diff-7d25cea7df3c99710f9ba35f5231579dcd76cebc8effb6fb0ea01dfa531e2b57R314 for all formats anyway.